CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - changed the way music timestamps are internally handled – resets music position in existing saves
 - changed vertex and fragment shaders into unified files that are runtime pre-processed for OpenGL versions 2.1 or 3.3
 - fixed a missing translation for the Spanish config tool for the Eidos logo skip option (#1151)
+- added Italian localization to the config tool
```

Comment:
Sorry to nitpick, but could you move the entry above so it comes after `added optional support for OpenGL 3.3 Core Profile`? This keeps all the `added`, `changed` etc entries grouped together.

Reply:
No problem